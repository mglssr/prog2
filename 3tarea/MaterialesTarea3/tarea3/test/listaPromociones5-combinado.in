crearListaPromociones
esVaciaListaPromociones
crearPromocion 01/01/2022 01/07/2022 10 1
agregarAPromocion 9 0 1 2 3 4 5 6 7 8
agregarAListaPromociones
crearPromocion 01/02/2022 01/06/2022 10 2
agregarAPromocion 9 0 1 2 3 4 5 6 7 8
agregarAListaPromociones
crearPromocion 02/03/2022 01/05/2022 10 3
agregarAPromocion 9 0 1 2 3 4 5 6 7 8
agregarAListaPromociones
imprimirListaPromociones
crearPromocion 02/03/2023 01/07/2023 10 3
agregarAPromocion 9 0 1 2 3 4 5 6 7 8
esCompatibleListaPromociones
liberarPromocion
unirListaPromociones 2 05/03/2002 01/07/2002 10 5 2 3 4 04/03/2002 01/07/2002 10 4 2 2 3
esVaciaListaPromociones
pertenecePromocionListaPromociones 2
obtenerPromocionListaPromociones 3
obtenerPromocionesFinalizadas 02/05/2022
obtenerPromocionesActivas 02/01/2022
imprimirListaPromociones
liberarListaPromociones
Fin
