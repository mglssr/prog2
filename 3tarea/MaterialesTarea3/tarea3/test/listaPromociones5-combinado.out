Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Lista de promociones creada con exito.
2> La lista de promociones es vacia.
3> Promocion creada con exito.
4> Productos agregados a la promocion de forma exitosa.
5> Se ha agregado la promocion a la lista de forma exitosa.
6> Promocion creada con exito.
7> Productos agregados a la promocion de forma exitosa.
8> Se ha agregado la promocion a la lista de forma exitosa.
9> Promocion creada con exito.
10> Productos agregados a la promocion de forma exitosa.
11> Se ha agregado la promocion a la lista de forma exitosa.
12> Promocion #1 del 1/1/2022 al 1/7/2022
Productos: 0 1 2 3 4 5 6 7 8 
Promocion #2 del 1/2/2022 al 1/6/2022
Productos: 0 1 2 3 4 5 6 7 8 
Promocion #3 del 2/3/2022 al 1/5/2022
Productos: 0 1 2 3 4 5 6 7 8 
13> Promocion creada con exito.
14> Productos agregados a la promocion de forma exitosa.
15> La promocion es compatible con el resto de las promociones de la lista.
16> Promocion liberada con exito.
17> Promocion #4 del 4/3/2002 al 1/7/2002
Productos: 2 3 
Promocion #5 del 5/3/2002 al 1/7/2002
Productos: 3 4 
Promocion #1 del 1/1/2022 al 1/7/2022
Productos: 0 1 2 3 4 5 6 7 8 
Promocion #2 del 1/2/2022 al 1/6/2022
Productos: 0 1 2 3 4 5 6 7 8 
Promocion #3 del 2/3/2022 al 1/5/2022
Productos: 0 1 2 3 4 5 6 7 8 
18> La lista de promociones NO es vacia.
19> La promocion de id 2 pertenece a la lista de promociones.
20> Promocion #3 del 2/3/2022 al 1/5/2022
Productos: 0 1 2 3 4 5 6 7 8 
21> Promociones finalizadas: 
Promocion #3 del 2/3/2022 al 1/5/2022
Productos: 0 1 2 3 4 5 6 7 8 
22> Promociones activas: 
Promocion #1 del 1/1/2022 al 1/7/2022
Productos: 0 1 2 3 4 5 6 7 8 
23> Promocion #2 del 1/2/2022 al 1/6/2022
Productos: 0 1 2 3 4 5 6 7 8 
24> Lista promociones liberada con exito.
25> Fin.
